WEP0000_wep-proposal.rst

```diff
@@ -0,0 +1,227 @@
+
+#. `Implementation`_ — If the WEP is accepted, the processes or policies described in the WEP are implemented by the CoreOps team, the WEP author, or specific community members as defined in the WEP.
+
+The following sections describe each stage in more detail.
+
+Pre-proposal
+............
+
+The WEP process begins with a new idea for Write the Docs. It is highly recommended that a single WEP contain a single key proposal or new idea. 
+
+To help prepare the WEP draft, we encourage community members to discuss the idea publicly, for example in the Slack group, and gather initial community feedback before writing a full WEP. Public discussions on pre-proposals also help to make sure the idea is applicable to the entire community and not just to the author.
+
+A WEP can be co-authored by more than one community member, but we recommend keeping the number of co-authors small and choosing a representative who will submit the PR to the WEPs repository on behalf of the co-authors.
+
+Draft submission and pre-review
+...............................
+
+A draft WEP must be submitted in a GitHub pull request to the `writethedocs/weps <https://github.com/writethedocs/weps>`_ repository, in the ``drafts/`` directory. If you need help with working with GitHub or creating a PR, please feel free to send an e-mail to support@writethedocs.org, and the Write the Docs team will help you get started.
+
+The WEP draft must follow these guidelines:
+
+- WEP is actionable and implementable. The proposed change must be specific and the resulting action clear.
+- WEP fits the scope of the WEP process, as described in **Scope of WEPs**. WEPs that are out of scope of the process will be rejected by the reviewer.
+- Content format must follow the WEP style as described in the **WEP format and style** section. This includes language, grammar, structure, and markup. The PR might be blocked until proper formatting rules are applied.
+- The WEP content is complete. If you want to submit a Work in Progress (WIP) draft before it is ready for pre-review, you can mark the PR as ``WIP`` and the review team will wait until the draft is ready.
+- **Allow edits from maintainers** option is selected. This helps the reviewer merge your WEP draft if it is accepted.
+- The PR is tagged with a **draft** flag, to indicate that it is in a draft state and is ready for pre-review. 
+
+
+After the pull request for the WEP draft is submitted, a WEP team member reviews the pull request to make sure it adheres to the guidelines. If the WEP is clearly not ready, the reviewer might reject the pull request and ask the author to submit a new WEP after the problems have been fixed.
```

Comment:
This seems redundant to line 117

Reply:
I think it's trying to describe the process the WEP team will take, instead of defining the rules of the process itself.